pkg/planner/core/memtable_predicate_extractor.go

```diff
@@ -1825,7 +1826,8 @@ func (e *InfoSchemaTablesExtractor) Extract(ctx base.PlanContext,
 	predicates []expression.Expression,
 ) (remained []expression.Expression) {
 	var resultSet, resultSet1 set.StringSet
-	e.colNames = []string{"table_schema", "constraint_schema", "table_name", "constraint_name", "sequence_schema", "sequence_name", "partition_name", "schema_name", "index_name"}
+	e.colNames = []string{"table_schema", "constraint_schema", "table_name", "constraint_name",
```

Comment:
we don't need to use a struct field `e.colNames`. We can use a global variable instead